README.md

```diff
@@ -30,22 +30,22 @@ flowchart LR
     commit -->|Finally|creatpr((Create a Pull Request))
     
  ```
-1. Fork the project:
+**1. Fork the project :**
```

Comment:
I think it looks better without space before `:` (same for all the other instances).
```suggestion
**1. Fork the project:**
```